### ISSUE TITLE
import Pkg to move before Pkg.activate(".")

### ISSUE THREAD

**Issue by alaindanet** [BecksLab/JuliaTutorials_BecksLab#2]
`Pkg.activate(".")`option resulted in an error without running `using Pkg`.

Should we move this command a bit earlier in the doc?

**Comment by TanyaS08** [comment]
I don't think this is an issue anymore - at least in terms of website builds. I will keep this open for now but I think this is a stale issue
